tests/quantile.rs

```diff
@@ -373,16 +369,8 @@ impl<'a, R: Rng + 'a> Iterator for RandomMaxIter<'a, R> {
 
 /// Calculate the count at a quantile with arbitrary precision arithmetic
 fn calculate_quantile_count(quantile: f64, count: u64) -> u64 {
-    let product = Rational::from_f64(quantile).unwrap() * Rational::from(Integer::from(count));
-    let prod_as_int = (product).to_integer();
-
-    // emulate ceil()
-    if product == Rational::from(prod_as_int.clone()) {
-        return prod_as_int.to_u64().unwrap();
-    } else {
-        // to_integer()'s rounding down has chopped off a fractional part
-        return prod_as_int.to_u64().unwrap() + 1;
-    }
+    let product = Rational::from_f64(quantile).unwrap() * Rational::from(count);
+    product.ceil().to_u64().unwrap()
```

Comment:
@marshallpierce I know you've had a fair bit of time to play with these corner cases. Do you think rug's `ceil` does what you expect?

Reply:
It looks fine; I can't recall why I avoided using it to begin with.